src/cluster.c

```diff
@@ -942,7 +943,7 @@ clusterNode *createClusterNode(char *nodename, int flags) {
     node->configEpoch = 0;
     node->flags = flags;
     memset(node->slots,0,sizeof(node->slots));
-    node->slots_info = NULL;
+    node->slot_info_pair = NULL;
```

Comment:
I think a plural is good naming style for a list.

```suggestion
    node->slot_info_pairs = NULL;
```

Reply:
Agreed.